calculated_policies/aws_ec2_public_subnet/main.tf

```diff
@@ -19,35 +19,38 @@ resource "turbot_policy_setting" "instance_subnet_" {
   # GraphQL to pull instance tags
   template_input = <<EOT
   {
-    resource {
-      subnetId: get(path: "SubnetId")
-    }
-    resources(filter:"resourceType:'tmod:@turbot/aws-vpc-core#/resource/types/routeTable'") {
-      items {
-        associations: get(path: "Associations.[0].SubnetId")
-        routes: get(path: "Routes")
+      resource {
+          subnetId: get(path: "SubnetId")
+      }
+      resources(filter:"resourceType:'tmod:@turbot/aws-vpc-core#/resource/types/routeTable'") {
+          items {
+              associations: get(path: "Associations")
+              routes: get(path: "Routes")
+          }
       }
-    }
   }
   EOT
 
   # Nunjucks Template
   template = <<EOT
+  {%- set regExp = r/^igw/g %}
   {%- set hasIGW = false -%}
   {%- for item in $.resources.items -%}
-    {%- if item.associations == $.resource.subnetId -%}
-      {%- for gateway in item.routes -%}
-        {%- if 'igw' in gateway.GatewayId -%}
-          {%- if hasIGW == false -%}
-            "Not approved"
-            {%- set hasIGW = true -%}
-          {%- endif -%}
+      {% for subnetid in item.associations %}
```

Comment:
Should be:

for association in associations. 

Can you fix things like that on this PR and then ping me and we can merge it immediately

Reply:
where does the associations come from? dont i need to use the item.associations format?